README.md

```diff
@@ -103,7 +103,7 @@ sudo easy_install pip && sudo pip install virtualenv
 ##### Ubuntu:
 ```
 sudo apt-get install build-essential git-core libgmp3-dev graphicsmagick  python-virtualenv
-python-dev docker-ce
+python-dev docker-ce pkg-config libssl-dev
```

Comment:
We had a script that checked for installed dependencies at one point, I thought it was [check.sh](https://github.com/mozilla/fxa/blob/c4706afc094f64ea1c20629d30ae4bbc6d19e6a8/_scripts/check.sh#L1). It looks like OSX already checks for gmp and graphicsmagick, I wonder if we could do the same with Linux dependencies.

Reply:
Think I'll merge this and file a follow up to enhance that script. There should be something with apt that'll be handy to check thing